### PR TITLE
Update @font-face page

### PR DESCRIPTION
**Pull request by hohoDigidea**
### Description

Updated and detailed the introduction of `local()`, `format()` and `tech()` in the `src` parameter according to the specification.

### CODE REVIEW

File: files/en-us/web/css/@font-face/index.md [hunk 1]
```diff
@@ -22,9 +22,10 @@ The **`@font-face`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At
 
 ```css
 @font-face {
-  font-family: "Open Sans";
-  src: url("/fonts/OpenSans-Regular-webfont.woff2") format("woff2"), url("/fonts/OpenSans-Regular-webfont.woff")
-      format("woff");
+  font-family: "Trickster";
+  src: local("Trickster"),
+       url("trickster-COLRv1.otf") format("opentype") tech(color-COLRv1),
+       url("trickster-outline.otf") format("opentype");
```
Comment (estelle) [inline files/en-us/web/css/@font-face/index.md:28]:
```suggestion
       url("trickster-outline.otf") format("opentype"),
       url("trickster-outline.woff") format("woff");
```

File: files/en-us/web/css/@font-face/index.md [hunk 2]
```diff
@@ -56,29 +57,41 @@ The **`@font-face`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At
 - {{cssxref("@font-face/size-adjust", "size-adjust")}}
   - : Defines a multiplier for glyph outlines and metrics associated with this font. This makes it easier to harmonize the designs of various fonts when rendered at the same font size.
 - {{cssxref("@font-face/src", "src")}}
-
-  - : Specifies the resource containing the font data. This can be a URL to a remote font file location or the name of a font on the user's computer.
-
-    To provide the browser with a hint as to what format a font resource is — so it can select a suitable one — it is possible to include a format type inside a `format()` function:
+  - : Specifies font resources. Is a comma-separated list representing the resource fallback order, each component containing a `url()` or a `local()`. `url()` can be followed by `format()` and `tech()`, like this:
```
Comment (estelle) [inline files/en-us/web/css/@font-face/index.md:60]:
```suggestion
  - : Specifies font resources. A comma-separated list representing the resource fallback order, each font file within  a `url()` or `local()`.  The `url()` can be followed by `format()` and `tech()`, like this:
```

File: files/en-us/web/css/@font-face/index.md [hunk 3]
```diff
@@ -56,29 +57,41 @@ The **`@font-face`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At
...
 
     ```css
-    src: url(ideal-sans-serif.woff) format("woff"), url(basic-sans-serif.ttf)
-        format("truetype");
+    src: local("Trickster"),
+         url("trickster-COLRv1.otf") format("opentype") tech(color-COLRv1),
+         url("trickster-outline.otf") format("opentype");
```
Comment (estelle) [inline files/en-us/web/css/@font-face/index.md:65]:
```suggestion
         url("trickster-outline.otf") format("opentype"),
         url("trickster-outline.woff") format("woff");
```

File: files/en-us/web/css/@font-face/index.md [hunk 4]
```diff
@@ -56,29 +57,41 @@ The **`@font-face`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At
...
     ```
 
-    The available types are: `"woff"`, `"woff2"`, `"truetype"`, `"opentype"`, `"embedded-opentype"`, and `"svg"`.
+    `local()`: Specifies the font name to not download the font when a local font is available. The font name may or may not be enclosed in quotes.
```
Comment (estelle) [inline files/en-us/web/css/@font-face/index.md:68]:
```suggestion
    `local()`: Specifies the font name for when the font is available in the user's operating system. Quoting the font name is optional.
```

File: files/en-us/web/css/@font-face/index.md [hunk 5]
```diff
@@ -56,29 +57,41 @@ The **`@font-face`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At
...
+
+    `format()`: **Optional**. Specify the font format. If the value is invalid, the browser may not download the resource; but if this parameter is not provided, the browser will always download the resource and then detect the format. The value can be a _keyword_ or a _string_, _keyword_ mean no quotes, _string_ require quotes. _keyword_ is newer specification, so perhaps not yet widely supported. _keyword_ support: `collection`, `embedded-opentype`, `opentype`, `svg`, `truetype`, `woff`, `woff2`. See below for supported _string_.
```
Comment (estelle) [inline files/en-us/web/css/@font-face/index.md:70]:
```suggestion
    `format()`: **Optional**. Specifies the font format. If the value is not supported or invalid, the browser may not download the resource, potentially saving bandwidth. If omitted, the browser will always download the resource and then detect the format. The value can be a _keyword_ or a _string_, (_strings_ require quotes, _keywords_ do not). Keywords support include  `collection`, `embedded-opentype`, `opentype`, `svg`, `truetype`, `woff`, `woff2`. See the table below for supported _string_ values.
```

File: files/en-us/web/css/@font-face/index.md [hunk 6]
```diff
@@ -56,29 +57,41 @@ The **`@font-face`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At
...
+
+    `tech()`: **Optional**. Not yet widely supported. Value is a _keyword_, support: `variations`, `palettes`, `incremental`, `features-opentype`, `features-aat`, `features-graphite`, `color-COLRv0`, `color-COLRv1`, `color-SVG`, `color-sbix`, `color-CBDT`.
+
+    The following are _string_ values supported by `format()`, and their new syntax equivalents:
+
+    | String form                     | Equivalent syntax                   |
+    | ------------------------------- | ----------------------------------- |
+    | `format("woff2")`               | `format(woff2)`                     |
```
Comment (estelle) [inline files/en-us/web/css/@font-face/index.md:78]:
```suggestion
    | `format("svg")`               | `format(svg)`                     |
    | `format("woff2")`               | `format(woff2)`                     |
```

File: files/en-us/web/css/@font-face/index.md [hunk 7]
```diff
@@ -56,29 +57,41 @@ The **`@font-face`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At
...
 - {{cssxref("@font-face/unicode-range", "unicode-range")}}
   - : The range of Unicode code points to be used from the font.
 
 ## Description
 
-If the `local()` function is provided, specifying a font name to look for on the user's computer, and if the {{Glossary("user agent")}} finds a match, that local font is used. Otherwise, the font resource specified using the `url()` function is downloaded and used.
-
 By allowing authors to provide their own fonts, `@font-face` makes it possible to design content without being limited to the so-called "web-safe" fonts (that is, the fonts which are so common that they're considered to be universally available). The ability to specify the name of a locally-installed font to look for and use makes it possible to customize the font beyond the basics while making it possible to do so without relying on an Internet connection.
 
-It's common to use both `url()` and `local()` together, so that the user's installed copy of the font is used if available, falling back to downloading a copy of the font if it's not found on the user's device.
```
Comment (estelle) [inline files/en-us/web/css/@font-face/index.md:80]: same comment as above.

**Review by estelle — COMMENTED** [review]
made some suggestions and a few questions

**Comment by LIXiangChen** [comment]
@estelle I brought back the two deleted paragraphs, but their meaning is a bit repetitive and I don't know how to make it better, maybe you can edit it. And after applying your suggested changes, I made some additions.

File: files/en-us/web/css/@font-face/index.md [hunk 8]
```diff
@@ -56,28 +58,65 @@ The **`@font-face`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At
...
-  - : Specifies the resource containing the font data. This can be a URL to a remote font file location or the name of a font on the user's computer.
+    ```css
+    src: local("Trickster"),
+         url("trickster-COLRv1.otf") format("opentype") tech(color-COLRv1),
+         url("trickster-outline.otf") format("opentype"),
+         url("trickster-outline.woff") format("woff");
+    ```
 
-    To provide the browser with a hint as to what format a font resource is — so it can select a suitable one — it is possible to include a format type inside a `format()` function:
+    `url()`: Specifies the URL of a font file, like any other `url()` in CSS. If the font file used as a container contains multiple fonts, one of them should be specified, as follows:
```
Comment (estelle) [inline files/en-us/web/css/@font-face/index.md:70]:
```suggestion
    `url()`: Specifies the URL of a font file, like any other `url()` in CSS. If the font file is a container for multiple fonts, a fragment identifier is included to indicate which sub-font should be used, as follows:
```

File: files/en-us/web/css/@font-face/index.md [hunk 9]
```diff
@@ -56,28 +58,65 @@ The **`@font-face`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At
...
 
     ```css
-    src: url(ideal-sans-serif.woff) format("woff"), url(basic-sans-serif.ttf)
-        format("truetype");
+    src: url(collection.otc#WhichFont); /* WhichFont is the PostScript name of a font in the font file */
+    src: url(fonts.svg#WhichFont);      /* WhichFont is the element id of a font in the SVG Font file */
     ```
 
-    The available types are: `"woff"`, `"woff2"`, `"truetype"`, `"opentype"`, `"embedded-opentype"`, and `"svg"`.
+    `local()`: Specifies a font name for when the font is available in the user's device. Quoting the font name is optional.
```
Comment (estelle) [inline files/en-us/web/css/@font-face/index.md:77]:
```suggestion
    `local()`: Specifies the font name should the font be available on the user's device. Quoting the font name is optional.
```

**Review by estelle — APPROVED** [review]
thank you.

**Pull request merged**